### PR TITLE
Create knex clients seed

### PR DESCRIPTION
**Pull request by eliasfang**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Referenced doc: https://gist.github.com/NigelEarle/70db130cc040cc2868555b29a0278261
- Ran knex seed:make 01_clients to create seeds/01_clients.js
- Changed table_name to 'clients' to match clients table
- Filled in test data for id, first_name, last_name, email, is_recicipient
- Ran knex seed:run to seed data into database

### Testing
How did you confirm your changes worked? 
- Confirmed that all data was seeded into clients table in database on DataGrip (see below)
- Checked that seeded data matched data in 01_clients.js

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Seeded 'clients' Table | 01_clients.js Data |
| --- | --- |
| image | image |

### CODE REVIEW

File: app/database/seeds/01_clients.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('clients').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('clients').insert([
+        {
+          id: 1,
```
Comment (mingkaili) [inline app/database/seeds/01_clients.js:8]: I think each clients also needs a foreign key to the jobs table

Comment (eliasfang) [reply app/database/seeds/01_clients.js:8]: Added and updated Admin Info!

**Review by mingkaili — APPROVED** [review]
Thanks, LGTM

File: app/database/seeds/01_clients.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          first_name: 'Elias',
+          last_name: 'Fang',
+          email: 'efang@ucsd.edu',
+          is_recipient: true,
+          job_id: 6
```
Comment (shravan1k) [inline app/database/seeds/01_clients.js:13]: Are there jobs seeds as well for the job_id to map to?

Comment (eliasfang) [reply app/database/seeds/01_clients.js:13]: I'm not sure if I'm understanding your question but the job_id field should correspond to the id field in the 'jobs' table.

Comment (shravan1k) [reply app/database/seeds/01_clients.js:13]: Yeah are those jobs being added to the table?

Comment (eliasfang) [reply app/database/seeds/01_clients.js:13]: Yes

**Review by jacobayu — APPROVED** [review]
lgtm!

**Pull request merged**